keyboards/naked60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,151 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+  _QWERTY = 0,
+  _FLOCK,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  FLOCK = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  RGBRST
+};
+
+enum tapdances{
+  TD_ESFL = 0,
+  TD_ESQW,
+};
+
+// Layer Mode aliases
+#define _____ KC_TRNS
+#define XXXXX KC_NO
```

Comment:
QMK defines seven-character aliases for these.

https://github.com/qmk/qmk_firmware/blob/c2709a7ca4a139af0606b085d21a0af460dbf87e/quantum/quantum_keycodes.h#L29-L31

If you use the QMK defaults, you don't need this block. This goes for all of your `keymap.c` files.

Reply:
Thank you.
I changed all the keymaps.